### PR TITLE
Fix Github Actions CI Script

### PR DESCRIPTION
**Pull request by sunkickr**
Fixes github actions CI/CD script and applies suggestion from the issue below:

Issue:
- https://github.com/astronomer/docs/issues/483

### CODE REVIEW

**Review by jwitz — APPROVED** [review]
LGTM!

File: astro/ci-cd.md [hunk 1]
```diff
@@ -66,8 +66,9 @@ To automate code deploys to a Deployment using [GitHub Actions](https://github.c
 
    - `ASTRONOMER_KEY_ID` = `<your-key-id>`
    - `ASTRONOMER_KEY_SECRET` = `<your-key-secret>`
+   - `DEPLOYMENT_ID` = `<your-astro-deployment-id>`
 
-2. Add the following to a new file in `.github/workflows`, making sure to replace `<organization-id>` and `<deployment-id>` with the values for your Deployment:
+2. Add the following to a new file in .github/workflows, for example `cicd.yaml`:
```
Comment (jwitz) [inline astro/ci-cd.md:71]:
```suggestion
2. Create a new YAML file in `.github/workflows` that includes the following configuration:
```

File: astro/ci-cd.md [hunk 2]
```diff
@@ -66,8 +66,9 @@ To automate code deploys to a Deployment using [GitHub Actions](https://github.c
 
    - `ASTRONOMER_KEY_ID` = `<your-key-id>`
    - `ASTRONOMER_KEY_SECRET` = `<your-key-secret>`
+   - `DEPLOYMENT_ID` = `<your-astro-deployment-id>`
```
Comment (paolaperaza) [inline astro/ci-cd.md:69]:
```suggestion
   - `ASTRONOMER_DEPLOYMENT_ID` = `<your-astro-deployment-id>`
```

File: astro/ci-cd.md [hunk 3]
```diff
@@ -87,11 +88,11 @@ To automate code deploys to a Deployment using [GitHub Actions](https://github.c
...
         - name: checkout repo
-        uses: actions/checkout@v2.3.4
+          uses: actions/checkout@v2.3.4
         - name: Deploy to Astro
-        run: |
-          brew install astronomer/cloud/astrocloud
-          astrocloud deploy <deployment-id>
+          run: |
+            brew install astronomer/cloud/astrocloud
+            astrocloud deploy ${{ secrets.DEPLOYMENT_ID }}
```
Comment (paolaperaza) [inline astro/ci-cd.md:95]:
```suggestion
            astrocloud deploy ${{ secrets.ASTRONOMER_DEPLOYMENT_ID }}
```

**Review by paolaperaza — APPROVED** [review]

**Comment by paolaperaza** [comment]
@sunkickr I made a suggestion to specify `ASTRONOMER` as part of the Deployment ID -- lmk what you think, but either way let's get this merged 😎

**Pull request merged**